src/deep-db/lib/Local/Driver/LocalDynamo.js

```diff
@@ -35,21 +52,32 @@ export class LocalDynamo extends PathAwareDriver {
   _start(cb) {
     let cbTriggered = false;
 
+    // avoid local-dynamo package issues...
+    fse.ensureDirSync(this.path);
+
     this._options.dir = this.path;
 
-    this._process = LocalDynamoServer.launch(this._options, this.port);
+    try {
+      this._process = LocalDynamoServer.launch(this._options, this.port);
+    } catch (error) {
+      this._pickUpOldInstance
+        ? cb(null)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/MitocGroup/deep-framework/file/1339245290/issues/source?bid=2888470&fileBranchId=2957139#l65)